presto-geospatial/src/main/java/com/facebook/presto/geospatial/GeoFunctions.java

```diff
@@ -53,9 +53,12 @@
 import static com.facebook.presto.spi.StandardErrorCode.INVALID_FUNCTION_ARGUMENT;
 import static com.facebook.presto.spi.type.StandardTypes.DOUBLE;
 import static com.google.common.base.Preconditions.checkArgument;
+import static java.lang.String.format;
 
 public final class GeoFunctions
 {
+    private static final Joiner COMMA_JOINER = Joiner.on(" or ");
```

Comment:
This isn't a comma. I'd just inline this since it's only used for error messages.